src/components/types/Date.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import Datetime from 'react-datetime';
+import './Date.css';
```

Comment:
Import the JS string of CSS here and use `insert-css` NPM module to load CSS.
If I doesn't work, try to do that in [`DOMContentLoaded`](https://developer.mozilla.org/en/docs/Web/Events/DOMContentLoaded) event.